documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -3,14 +3,8 @@
 ## Version 9.8.0
 
 ### New features
-* 
-
-### Changed features
-*
 
 ### Resolved issues
-* 
-
-### Dependency updates
-*
+* (IDETECT-4315) A filter was added to prevent performance issues related to the [company_name] [solution_name] API call that retrieves role information on startup.
+* (IDETECT-4360) Resolved an issue with component location analysis failing in some instances while running in Rapid scan mode with a [blackduck_product_name] Vulnerability Policy condition set to an overall score greater than 4.
```

Comment:
This eventually turned out to be not specific to Rapid mode and the error was reproduced in non-rapid mode with no policies.

To illustrate, when a line like `Abc {def} ghi ${jkl}` was encountered component locator attempted to extract substring from character 16 to 8, resulting in an index out of bounds exception. Now in cases like this it attempts to extract substring from character 16 to 19.

Reply:
@andrian-sevastyanov Thank you. I updated the note to be more generic.